src/xml/XmlStreamReader.cpp

```diff
@@ -187,7 +187,7 @@ stdcxx::optional<unsigned long> XmlStreamReader::getOptionalAttributeValue(const
 }
 
 template <>
-stdcxx::optional<std::string> XmlStreamReader::getOptionalAttributeValue(const std::string& attributeName) const {
+IIDM_DECLSPEC stdcxx::optional<std::string> XmlStreamReader::getOptionalAttributeValue(const std::string& attributeName) const {
```

Comment:
Here, `getOptionalAttributeValue` does not have any generic implementation in header file (or hxx file), the template specialization is in cpp file, so calling getOptionalAttribute<int>() cannot instanciate template because implementation is not provided, which would conflict with the generic+specialized one.
Hope I'm clear...